### PR TITLE
Grid but no alignment

### PR DESCRIPTION
**Pull request by Hoobachube**

### CODE REVIEW

File: grid.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            padding: 10px;
+            border: 1px solid black;
+            height:120px;
+            width: 120px;
+            float: left;
+        }
+        .row {
+            clear: both;
+        }
+    </style>
```
Comment (alexw1983) [inline grid.html]: There's more than one way to skin a cat and lots of different ways to achieve this. 
A few things to pick up on here:
1. This is fine for a demo but stylesheets should be in a separate file.
1. The .row div {} syntax affects all divs that are a children of the row element. That's fine and avoids having to put classes on all the sub divs. In this case though I would probably advise being more explicit. You might be adding extra divs you want to behave differently. Create a new class like .box or whatever and add it to the child divs. Note that this is just my opinion so feel free to ignore.
1. The clear both isn't really helping you.
1. To get it to appear centrally add a class to the outer div and give it the style of margin : 0 auto;

Here's what I had for this. Feel free to use it to compare. Don't feel you have to use this though.

`       
         .grid {
            margin: 0 auto;
            border: 1px solid black;
            width: 306px;
            height: 306px;
        }`
               `.box {
            border: 1px solid black;
            height: 100px;
            width: 100px;
            float: left;
        }`

File: grid.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    </head>
+<body>
```
Comment (alexw1983) [inline grid.html]: I'd treat yourself to an h1 tag here. and line up the text centrally

File: grid.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <div></div>
+        </div>
+
+        <div class="row">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+    </div>
```
Comment (alexw1983) [inline grid.html]: Here's what I had here to go with my styles above. Note that the row class isn't specifically styled above. That would be considered bad practice in most places but I have left it in for now to clearly show intention.


    <div class="grid">
        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>

        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>

        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>
    </div>

**Review by alexw1983 — CHANGES_REQUESTED** [review]
I have left some comments and a few code samples. Feel free to disagree with how I have approached it.

**Comment by alexw1983** [comment]
A good first attempt overall. I have come down quite a bit harder than I would in real life but that is just to try and give a bit more guidance. A few points on the whole thing overall.
1. I see why you used the grid.html file here but you can add this to the index.html file. By convention the root page of any website is index.html and this will be the root page of the application.
1. It is standard practice to name branches with the ticket number and separated by dashes. So in this case it would be 2-draw-grid. Other places might have a different convention but that is there problem.

File: grid.html (hunk 1 above)
Comment (Hoobachube) [reply grid.html]: I AM AWESOME

File: grid.html (hunk 3 above)
Comment (Hoobachube) [reply grid.html]: done

**Review by alexw1983 — APPROVED** [review]

**Pull request merged**